RELEASE_NOTES.md

```diff
@@ -37,6 +37,49 @@ You'll need to add `tailwind.config.js` to your app `.gitignore`. If you generat
 
 You can read more about this change on PR [\#9480](https://github.com/decidim/decidim/pull/9480).
 
+### 3.2 Change Webpacker to Shakapacker
+
+Since the Rails team has retired the Webpacker in favour or importmap-rails or js-bundling, we got ouserlves in a situation where performance improvements could not be performed.
+In order to continue having support for Webpacker like syntax, we have switched to Shakapacker.
+
+In order to perform the update, you will need to make sure that you **do not have webpacker in your Gemfile**.
+If you have it, please remove it, adn allow Decidim to handle the webpacker / shackapacker dependency.
+
+In order to perform the migration to shakapacker, please backup the following files, to make sure that you save any customizations you may have done to webpacker:
+
+```console
+config/webpacker.yml
+config/webpack/*
+package.json
+postcss.config.js
+```
+
+After all the backups and changes mentioned above have been completed, follow the default upgrade steps, as mentioned above in the document.
+Then run the below command, and replace all the configuration with the one that Decidim is providing by default:
+
+```console
+bundle exec rake decidim:webpacker:install
+```
+
+This will make the necessary changes in the `config/webpacker.yml`, but also in the `config/webpack/` folder.
+
+#### Note for development
```

Comment:
When I tested the "on-the-fly" compilation locally, I didn't see it being any slower than with webpacker. It was around 20s on my local machine with both webpacker and shakapacker.

But it is sure annoying that shakapacker prints out these warnings multiple times when it is enabled:

```
Prepare JS assets with either:
1. Running `bin/webpacker-dev-server`
2. Set `compile` to false in webpacker.yml and run `bin/webpacker -w`
[Webpacker] Everything's up-to-date. Nothing to do
Webpacker::Compiler - Slow setup for development
```

I didn't find any way to disable that during development because it is forced here for development environment:
https://github.com/shakacode/shakapacker/blob/4c92e2a956c443233a6f00b65df7db67b5cfb2ba/lib/webpacker/mtime_strategy.rb#L22-L30
https://github.com/shakacode/shakapacker/blob/4c92e2a956c443233a6f00b65df7db67b5cfb2ba/lib/webpacker/digest_strategy.rb#L32-L39

I actually liked this feature by default, so that you don't always have to start the `webpack-dev-server` for testing some PR for instance but it is annoying that we get these warnings printed into the console...

Also, do you happen to know why the CI is not timing out with webpacker as I believe we have that feature enabled?

Reply:
It seems that shakapacker is using mime strategy in development that will check for the files that needs to be compiled. Webpacked does not have that strategy. 

Also, the compilation is slow as currently the shakacpaker is using the default webpacker compile file. Once https://github.com/shakacode/shakapacker/pull/254 We could use the the file that decidim generates in order to serve assets.

Reply:
as discussed in maintainer meeting, we'll look into foreman or similar ( what rails uses internally )

Reply:
Overmind is pretty sweet because you can detach and debug your rails process.

Check out: https://github.com/shakacode/react_on_rails/blob/master/lib/generators/react_on_rails/bin/dev

Reply:
@justin808 thanks for your suggestion. We have chosen foreman, as it is part of rails stack. Decidim itself is already a huge pile of external libraries.